Week-3/Responsiveness/Bootstrap for a vacation website/script.js

```diff
@@ -0,0 +1,12 @@
+function showAlert(alertId) {
+   const bootstrapAlert = document.querySelector("#" + alertId);
+   const collapse = new bootstrap.Collapse(bootstrapAlert); 
+   collapse.show();
+}
+
+const modalYesBtn = document.querySelector("#contact-modal .btn-primary");
+modalYesBtn.addEventListener("click", function () {    
+   const sendBtn = document.querySelector("#send-message-btn");
```

Comment:
for this also you can use this document.getElementById

getElemetById is very very fast then querySelector

Reply:
This was the provided code on the assignments we just to need to make the changes in the HTML and CSS file